.github/ISSUE_TEMPLATE/bug-openapi-fetch.md

```diff
@@ -0,0 +1,42 @@
+---
+name: "Bug report: openapi-fetch"
+about: For the openapi-fetch library
+title: ""
+labels: openapi-fetch, bug
+assignees: ""
```

Comment:
I originally started to see if we could auto-assign to a specific team, but I think we need a custom action to do a round-robin assignment that I’ll add in a followup (assigning to the entire team means we all get notified, and I wanted to find a solution where we don’t all get notified for everything).

But in me examining the templates, I realized these were a little stale and could use some updating.